docs/csharp/programming-guide/interop/example-com-class.md

```diff
@@ -7,7 +7,7 @@ helpviewer_keywords:
 ms.assetid: 6504dea9-ad1c-4993-a794-830fec5270af
 ---
 # Example COM Class (C# Programming Guide)
-The following is an example of a class that you would expose as a COM object. After this code has been placed in a .cs file and added to your project, set the **Register for COM Interop** property to **True**. For more information, see [NIB: How to: Register a Component for COM Interop](http://msdn.microsoft.com/library/4de7d474-56e8-4027-994d-d47ca4725c5e).  
+The following is an example of a class that you would expose as a COM object. After this code has been placed in a .cs file and added to your project, set the **Register for COM Interop** property to **True**. For more information, see [NIB: How to: Register a Component for COM Interop](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/w29wacsy\(v=vs.100\)).
```

Comment:
nit unrelated to this PR, just caught my eye: the link text is not supposed to start with "NIB: "?

And question: should the link itself be as  
`https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/w29wacsy(v=vs.100)`  
without escaping parens at the end of the link (all other such links do not contain backslashes)? Now the link goes to 404

Reply:
Just noticed your question now @pkulikov. No, the NIB shouldn't be there but we have lots of them. Issue #1787 is tracking that. AFAIK the escaped parentheses don't cause issues in the rendered content but I've been using without escaping as well.

Reply:
I fixed the NIB. It was my understanding that the escapes for parens were necessary inside the link target so that the link will work in the HTML. Would be good to clarify that.

Reply:
@mikeblome cool, thank you
@mairaw thank you for the info; good to know that rendered content is without issues.